src/Microsoft.ML/Models/SaveAsOnnx.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Model.Onnx;
+
+namespace Microsoft.ML.Models
+{
+    /// <summary>
+    /// Converts a model to ONNX format.
+    /// </summary>
+    public sealed class SaveAsOnnx
+    {
+        /// <summary>
+        /// Converts and then saves a model to ONNX format.
+        /// </summary>
+        /// <param name="args">Arguments such as input model file path, output ONNX file path, etc.</param>
+        public static void Save(SaveOnnxCommand.Arguments args)
```

Comment:
>SaveOnnxCommand.Arguments [](start = 32, length = 25)

The way we solve the problem in invoking the runtime is to generate the C# API from entry-points (just as we do for other interfaces). Directly exposing the command-line support structures as the API interface is subideal.